ReSharper.FSharp/src/FSharp.Psi.Features/src/Daemon/Highlightings/CommonErrors.xml

```diff
@@ -360,4 +360,16 @@
     <QuickFix>RemoveRedundantQualifierFix</QuickFix>
   </Warning>
 
+  <Error staticGroup="FSharpErrors" name="NamespaceContainsValue">
```

Comment:
Please use the compiler error name.

Reply:
Also, the current name sounds like the namespace is the route of the error (which is wrong, since the error is about a value).